tests/rws_tests.py

```diff
@@ -276,6 +276,84 @@ def test_expected_rationales_case(self):
         loaded_sets = rws_check.load_sets()
         self.assertEqual(rws_check.error_list, [])  
 
+class TestCheckAssociatedCount(unittest.TestCase):
+    def test_within_limit(self):
+        json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://primary.com",
+                    "associatedSites": ["https://associated1.com"],
+                    "rationaleBySite": {}
+                }
+            ]
+        }
+        rws_check = RwsCheck(rws_sites=json_dict,
+                      etlds=None,
+                       icanns=set())
+        loaded_sets = rws_check.load_sets()
+        rws_check.check_associated_count(loaded_sets)
+        self.assertEqual(rws_check.associated_warning, [])
+        
+    def test_over_limit(self):
+        json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://primary.com",
+                    "associatedSites": ["https://associated1.com",
+                                        "https://associated2.com",
+                                        "https://associated3.com",
+                                        "https://associated4.com",
+                                        "https://associated5.com",
+                                        "https://associated6.com"],
+                    "rationaleBySite": {}
+                }
+            ]
+        }
+        rws_check = RwsCheck(rws_sites=json_dict,
+                      etlds=None,
+                       icanns=set())
+        loaded_sets = rws_check.load_sets()
+        rws_check.check_associated_count(loaded_sets)
+        self.assertEqual(rws_check.associated_warning, 
```

Comment:
Please check that the warning we want is *among* the actual warnings; not that it is the *only* warning. (You could do this with `self.assertIn` or `self.assertCountEqual`.)

Same comment for the test case below.